README.md

```diff
@@ -121,3 +121,6 @@ Its nine tails stand for end-to-end multi-tenancy support of this project.
 ## License
 
 This project is licensed under the Apache 2.0 License. See the [LICENSE](./LICENSE) file for details.
+
+## Community
+ Join in our [Slack](https://join.slack.com/t/apachekyuubi/shared_invite/zt-1e1qw68g4-yE5HJsVVDin~ABtZISyuxg) Channel !
```

Comment:
nit: Add a new line between head and content , and remove the first space char

Reply:
This change has been removed and placed in the CONTRIBUTE file